assets/sass/components/global/_googlesitekit-chart.scss

```diff
@@ -53,6 +53,20 @@
 	}
 }
 
+.googlesitekit-chart--LineChart div.google-visualization-tooltip {
+
+	height: fit-content !important;
```

Comment:
As `!important` should only be used in exceptional cases, it would be nice to include a comment here to explain why it's needed, would you mind adding one?

Reply:
Totally, I agree and just forgot to do this one. Added! 👍🏻